packages/zustand-firestore/src/contexts/createDocumentSnapshotStoreContext.ts

```
@@ -1,6 +1,6 @@
 import { createContext } from 'react'
 
-import { type DocumentData } from 'firebase/firestore'
+import { type DocumentData } from '@firebase/firestore'
```

ESLint

[error] Unable to resolve path to module '@firebase/firestore'.

(import-x/no-unresolved)